src/main/java/mate/academy/internetshop/dao/jdbc/RoleDaoJdbcImpl.java

```diff
@@ -54,7 +54,7 @@ public Role getRoleByName(Role.RoleName roleName) {
         return null;
     }
 
-    public void deleteRoleToDB(Long userId) {
+    public void deleteRole(Long userId) {
```

Comment:
```suggestion
    public void deleteAllRolesForUser(User user) {
```

і вже в методі ти зможеш використати user.getId()